### DISCUSSION TITLE
Get all the characters

### DISCUSSION THREAD

**Discussion by JuanF2019** [danielschuster-muc/potter-db#745] [Q&A] [answered]
How do you get all of the characters?

When using a GET request to /characters you get the first 100 characters, how can you get the rest?

**Comment by danielschuster-muc** [comment, marked as answer]
This is done via pagination: `https://api.potterdb.com/v1/characters?page[number]=5`. You can also specify the number of results per page with `page[size]=50` etc. (100 is the limit however). 
I should probably add a note in the documentation.
